src/NetAppFiles/NetAppFiles/ChangeLog.md

```diff
@@ -18,6 +18,12 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 0.1.2
```

Comment:
@leonardbf you can remove this version header and put the below content underneath the `Upcoming Release` header -- when a new version of the module should be released, we will create a separate PR that does the necessary versioning for this module to get it ready for release.

Reply:
ok, updated.